### PR TITLE
doc (CONTRIBUTION): Add missing English translations and clearify dns

### PR DESCRIPTION
**Pull request by Maledong**
1) Due to different service providors have a variety ways of coping with
a dns that cannot be analysed, we have to make it clear to users that
dns unit tests may fail (See: #3033).

2) Comment out a skip unit test in 'httpclient.test.js', no need to show
the 'pending' state but with comments above.

3) Add missing English part in CONTRIBUTING.md.

4) Change the warning info, because we can still use this callback but
NOT recommended.

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -159,14 +193,20 @@ In the release of every stable version, there will be a PM who has the following
...
     $ npm run commits
     ```
 - Nominate PM for next stable version.
 
 #### During Release
 
-All tags mentioned above refere to adding tags from npm in `package.json`.
+- Back up the stable version (master) onto the branch named after the current major (e.g: `1.x`), and set the tag to `release-{v}.x` (v is the current version like `release-1.x`).
+- Push the `next` branch to `master`, make it to the last stable one and remove `next` tag, change the
+contents corresponding to the branch in README.
```
Comment (killagu) [inline CONTRIBUTING.md]: Please remove the line break.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -159,14 +193,20 @@ In the release of every stable version, there will be a PM who has the following
...
+- Publish the latest stable version to [npm], and notify the previous framework to be upgraded.
```
Comment (killagu) [inline CONTRIBUTING.md]: There is no link named npm, please add it.

File: CONTRIBUTING.zh-CN.md [hunk 3]
```diff
@@ -51,14 +51,46 @@
 $ git checkout -b branch-name
 
 # 开发完成后跑下测试是否通过，必要时需要新增或修改测试用例
-$ npm test
+$ npm run test
```
Comment (killagu) [inline CONTRIBUTING.zh-CN.md]: > This runs a package's "test" script, if one was provided.

[`npm test`](https://docs.npmjs.com/cli/test) will execute the 'test' script. `run` is useless.

**Comment by Maledong** [comment]
@killagu：Please ignore this problem due to the issue of `egg-multipart` (Fix at: https://github.com/eggjs/egg-multipart/pull/21, not released yet).

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -55,6 +55,13 @@ $ git commit -m "fix(role): role.use must xxx"
 $ git push origin branch-name
 ```
 
+**Notice: In your local machine you may meet some unit tests that aren't passed after finishing running all of them. They are related to 'dns' problem. These cases are:**
```
Comment (dead-horse) [inline CONTRIBUTING.md]: 这个问题已经修复了吧？

Comment (Maledong) [reply CONTRIBUTING.md]: 没有。在一些机器上仍然有此类问题。所以我需要补充一句——这问题分两层说：
1）那个fmerk的修复是对的。
2）但是有些机器仍然有这些问题，比如我的另外一个联通的网络还是存在。所以这个还和网络配置和网络运行商相关，告知其它可以如测试仍然无法通过，可以忽略此问题。

Comment (Maledong) [reply CONTRIBUTING.md]: 最好是加上去吧？因为这个测试不是全部都通得过，以免他人引起误会，故此说明:)

File: test/lib/core/httpclient.test.js [hunk 5]
```diff
@@ -79,21 +79,23 @@ describe('test/lib/core/httpclient.test.js', () => {
...
-      assert(info.error.message.includes('mock res error'));
-      done();
-    });
-
-    client.request(url).catch(() => {
-      // it will print
-      // console.error(e.stack);
-    });
-  });
+  // This test is deprecated due to https://github.com/node-modules/urllib/pull/288
```
[the remark this answers is not in the document]
Comment (Maledong) [reply test/lib/core/httpclient.test.js]: ok

File: CONTRIBUTING.md (hunk 4 above)
Comment (dead-horse) [reply CONTRIBUTING.md]: 没必要，去掉就可以合并了

**Comment by Maledong** [comment]
@dead-horse : done :)

**Pull request merged**